### ISSUE TITLE
[Toc-ToGC] Twin Val'Kyr

### ISSUE THREAD

**Issue by alexuktc** [dalaranwow/dalaran-wow#4433]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Twin Val'Kyrs act like Mimiron. If you kill one of the Twins while the other is casting a shield (probably true for the Dark Vortex as well but we did not test it) the one casting the shield will remain alive and finish casting the shield (granted that the dps is not enough to pop it) and heal up. 
The portals were removed and no more orbs were spawning - meaning the triggers for the boss were removed, but the Valkyr was alive. Since we had the opposite color buff we did 50% less damage and hit the enrage timer (on HC). We still downed the boss but this is not normal. 

**Expected behaviour**: Tell us what should happen instead
No matter what the second one of the Twins dies the second should die as well. The only reason why Twins are tanked together is so that they can be cleaved for bonus damage.

**Steps to reproduce the problem**:

1. Have a GM start the encounter 
2. /kill The Twin not casting the shield
3. See that the other Twin remains alive.

**Include proofs for this behaviour**
I do not have any at the moment. I did not expect this to happen and did not have the time to be thinking of screenshots or anything else. It is easy enough to reproduce.

http://wowwiki.wikia.com/wiki/Twin_Val'kyr - Reading up on mechanics you will see that they share a health pool and one dying will kill the other as well. 

http://www.wowhead.com/npc=34497/fjola-lightbane#comments - Comments say they share a healthpool as well.

https://www.youtube.com/watch?v=jnq3ZduyHrk - Tankspot's guide to Twin Valkyr. ~2:30 he says they share a health pool

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
npc=34497
npc=34496

**Include Screenshots from the issue if necessary**
I don't have any at the moment. I hope not to need them.

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/4168
